xml/depl_poc.xml

```diff
@@ -694,7 +695,7 @@ The sizing recommendation includes an Admin Node (bare metal or VM), Controller
     </para>
   </note>
   <para>
-    <emphasis>Network mode</emphasis>. What mode to choose for a POC deployment depends on the High Availability (HA) requirements. The team network mode is required for an HA setup of &cloud;.
+    <emphasis>Network mode</emphasis>. What mode to choose for a Poc deployment depends on the High Availability (HA) requirements. The team network mode is required for an HA setup of &cloud;.
```

Comment:
PoC

Reply:
Thanks for the eagle eyes! In fact, found 3 appearances of 'Poc' in depl_poc.xml, corrected all of them now.